### DISCUSSION TITLE
v2.0.2 not loading after update

### DISCUSSION THREAD

**Discussion by vmpcee** [pear-devs/pear-desktop#1289] [General]
anyone else getting a blank page after updating? the addons and tweaks are there but there is no content, its just blank. i tried auto updating but after it said "complete" when i launched the app it did as said before. I also tried with scoop but got the same result even after fully uninstalling. any tips or solutions?

**Comment by vmpcee** [comment]
incase anyone finds this, a simple fix is to change the adblocker plugin to "In player" rather than "With blocklists"

**Comment by JellyBrick** [comment]
#1269
